lua/toggleterm/ui.lua

```diff
@@ -202,17 +202,29 @@ function M.switch_buf(buf)
   if cur_buf ~= buf then vim.cmd(fmt("keepalt buffer %d", buf)) end
 end
 
-local split_commands = {
-  horizontal = {
-    existing = "rightbelow vsplit",
-    new = "botright split",
-    resize = "resize",
-  },
-  vertical = {
-    existing = "rightbelow split",
-    new = "botright vsplit",
-    resize = "vertical resize",
-  },
+local function split_command(direction, existing)
+  if 'horizontal' == direction then
```

Comment:
It's clear to me if this condition is reversed, i.e.
```suggestion
  if direction == 'horizontal' then
```